aspnetcore/fundamentals/logging/index.md

```diff
@@ -897,6 +897,35 @@ The following example shows how to register filter rules in code:
 * Log level `Information` and higher.
 * All categories starting with `"Microsoft"`.
 
+::: moniker-end
+
+::: moniker range=">= aspnetcore-5.0"
+
+## Automatically log scope with SpanId, TraceId, and ParentId
+
+The logging libraries implicitly create a scope object with `SpanId`, `TraceId`, and `ParentId`. This behavior is configured via <xref:Microsoft.Extensions.Logging.LoggerFactoryOptions.ActivityTrackingOptions>.
+
+```csharp
+  var loggerFactory = LoggerFactory.Create(logging =>
+  {
+      logging.Configure(options =>
+      {
+          options.ActivityTrackingOptions = ActivityTrackingOptions.SpanId
```

Comment:
In .NET 6.0 we have added extra ActivityTrackingOptions values `Tags` and `Baggage`. do we want to include those too?

Reply:
Add the following two lines.

```C#
 | ActivityTrackingOptions.Baggage
 | ActivityTrackingOptions.Tags
```